core/src/main/java/org/mule/runtime/core/internal/profiling/DefaultProfilingService.java

```diff
@@ -193,4 +197,19 @@ private String getScope() {
       return getArtifactId(muleContext);
     }
   }
+
+  @Override
+  public ExportedSpanCapturer getExportedSpanCapturer() {
```

Comment:
I think we can improve the design here by adding an interface to the export of the traces.